### PR TITLE
More GUI stuff

### PR DESCRIPTION
**Pull request by maxpowa**
Highlights:

Fixed incorrect Gui.onMousePressed(game, mouseY, mouseButton) -> Gui.onScaleChange(game, windowWidth, windowHeight)
Added mapping for WidgetButton.onPressed / onReleased
Mapped most of GuiBackupLevel / GuiBackupPrompt

### CODE REVIEW

File: mappings/cko.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS cko
+	METHOD a rescale ()V
```
Comment (asiekierka) [inline mappings/cko.mapping]: This method is called on MinecraftGame to update the GUI scale, no? Shouldn't it then be updateScale or something in this vein?

File: mappings/cqn.mapping [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+			ARG 0 mouseX
+			ARG 1 mouseY
 	METHOD a draw (IIF)V
 		ARG 0 mouseX
 		ARG 1 mouseY
 		ARG 2 delta
-	METHOD b onMousePressed (Lclp;II)V
-		ARG 1 mouseY
-		ARG 2 mouseButton
+	METHOD b onScaleChange (Lclp;II)V
```
Comment (asiekierka) [inline mappings/cqn.mapping]: onScaleChanged, to match the others.

File: mappings/net/minecraft/client/gui/Gui.mapping [hunk 3]
```diff
@@ -18,6 +18,10 @@ CLASS cpt net/minecraft/client/gui/Gui
 		ARG 0 stack
 		ARG 1 x
 		ARG 2 y
+	METHOD a reinitialize (Lclp;II)V
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/Gui.mapping]: This should be initialize and the current init() most likely onInitialized?

File: mappings/net/minecraft/client/gui/menu/GuiBackupPrompt.mapping [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	METHOD <init> (Lcpt;Lcou$a;Ljava/lang/String;Ljava/lang/String;)V
+		ARG 0 parent
+		ARG 1 callback
+		ARG 2 title
+		ARG 3 subtitle
+	METHOD a draw (IIF)V
+		ARG 0 mouseX
+		ARG 1 mouseY
+		ARG 2 delta
+	METHOD ah_ canClose ()Z
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/menu/GuiBackupPrompt.mapping:32]: canClose or canBeClosed?

File: mappings/cko.mapping (hunk 1 above)
Comment (maxpowa) [reply mappings/cko.mapping]: yep, that's what its for. updateScale makes sense, will change to that

File: mappings/net/minecraft/client/gui/menu/GuiBackupPrompt.mapping (hunk 4 above)
Comment (maxpowa) [reply mappings/net/minecraft/client/gui/menu/GuiBackupPrompt.mapping:32]: I didn't map this, was preexisting from one of this classes ancestors.

Comment (asiekierka) [reply mappings/net/minecraft/client/gui/menu/GuiBackupPrompt.mapping:32]: Ah, sorry.

**Comment by maxpowa** [comment]
Updated as per review

**Pull request merged**